.buildkite/0_webui.yml

```diff
@@ -0,0 +1,14 @@
+agents:
+  queue: "juliaecosystem"
+  sandbox.jl: true
```

Comment:
As a reminder, it's best to put these as step-specific, in case we eventually want to have different kinds of agents for different steps in the future.

Reply:
Addressed in e2bfdab and d3d228f.